com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationArchiveExtractor.java

```diff
@@ -5,12 +5,25 @@
 import java.io.OutputStream;
```

Comment:
Stop using getNextEntryByName for modifying global content if possible. Please use the returned input stream.